src/providers/__tests__/googleResponse.json

```diff
@@ -70,7 +70,6 @@
         "amusement_park",
         "establishment",
         "point_of_interest",
-        "premise",
```

Comment:
I believe that this value is no longer a possible option in the Google response, should double-check though.